packer.py

```diff
@@ -0,0 +1,104 @@
+from copy import deepcopy
+import json
+from py3dbp import Item, Bin, Packer
+from rich import print
+from rich.tree import Tree
+
+def load_box_types(file="boxes.json"):
+    """Imports JSON file with definitions of boxes available."""
+    try:
+        with open(file) as f:
+            bins = json.load(f)
+    except FileNotFoundError:
+        print("File not found.")
+        return 1
+    return bins
+def load_items_types(file="items.json"):
+    """Imports items from the basket in JSON format"""
+    try:
+        with open(file) as f:
+            items = json.load(f)
+    except FileNotFoundError:
+        print("File not found.")
+        return 1
+    return items
+def create_items(items: dict) -> list:
+    item_list = []
+    for item in items:
+        for number in range(items[item]['quantity']):
+            item_list.append(Item(
+                items[item]['name'],
+                items[item]['width'],
+                items[item]['hight'],
+                items[item]['depth'],
+                items[item]['weight']))
+    return item_list
+def create_bins(bins: dict) -> list:
+    bins_list = []
+    for bin in bins:
+        bins_list.append(Bin(
+            bins[bin]['name'],    # Bin(name, width, height, depth, max_weight)
+            bins[bin]['width'],
+            bins[bin]['hight'],
+            bins[bin]['depth'],
+            bins[bin]['max_weight']))
+    return bins_list
+def refresh_items(unfitted_items):
+    for item in unfitted_items:
+        packer.add_item(item)
+
+                # Item(name, width, height, depth, weight)
+                # Item(items[item]['name'],
+                #      items[item]['width'],
+                #      items[item]['hight'],
+                #      items[item]['depth'],
+                #      items[item]['weight']))
+def refresh_bin_types(bin_types):
+    for bin in bin_types:
+        packer.add_bin(bin)
+def execute_packing(items_to_fit: list, visualize=True, export_img=False, textualize=True) -> list:
+    fitted_items = []   # list of solutions
+    tree = Tree("Packing list:", highlight=True, hide_root=True)
+    while items_to_fit:
+        packing_efficacy = 0
+
+        global packer
```

Comment:
Avoid global ! Better pass it as parameter, return it from function.

Reply:
Ok. Passed as parameter.